engine/src/main/cpp/krkr_bridge.cpp

```
@@ -277,17 +278,24 @@ bool isLaunchSceneReady(void* scene) {
     return formCount > 0;
 }
 
+
+bool startsWithPathPrefixCaseInsensitive(const char* str, const char* prefix, size_t prefixLen) {
+    if (str == nullptr || prefix == nullptr) return false;
+    if (strncasecmp(str, prefix, prefixLen) != 0) return false;
+    const char next = str[prefixLen];
+    return next == '\0' || next == '/';
+}
+
 bool pathMatchesPrefix(const char* path) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)